sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -119,7 +119,7 @@ object JdbcUtils extends Logging {
       isCaseSensitive: Boolean,
       dialect: JdbcDialect): String = {
     val columns = if (tableSchema.isEmpty) {
-      rddSchema.fields.map(x => dialect.quoteIdentifier(x.name)).mkString(",")
+      rddSchema.fields.map(x => dialect.quoteIdentifier(x.name.replace("\"", ""))).mkString(",")
```

Comment:
This will replace all the quotes. We just need to `stripPrefix` and `stripSuffix` of double quotes.

Reply:
Thanks for the review @gatorsmile , agreed. Will apply the changes and resend the updated PR soon.